docs/diagrams/StatusClassDiagram.puml

```diff
@@ -1,4 +1,8 @@
 @startuml
+!include style.puml
+skinparam classBackgroundColor MODEL_COLOR
+show members
+
 enum Status {
```

Comment:
do we need an ` <<enumeration>> ` label for the diagram?

Reply:
Oh yeah probably should add that